crates/packager/src/package/app/mod.rs

```diff
@@ -52,6 +52,77 @@ pub(crate) fn package(ctx: &Context) -> crate::Result<Vec<PathBuf>> {
 
     tracing::debug!("Copying frameworks");
     let framework_paths = copy_frameworks_to_bundle(&contents_directory, config)?;
+
+    // All dylib files and native executables should be signed manually
+    // It is highly discouraged by Apple to use the --deep codesign parameter in larger projects.
+    // https://developer.apple.com/forums/thread/129980
+    for framework_path in &framework_paths {
+        if let Some(framework_path) = framework_path.to_str() {
+            // Find all files in the current framework folder
+            let constructed_glob = format!("{}/**/*", framework_path);
+            let files = match glob::glob(&constructed_glob) {
+                Ok(res) => res,
+                Err(err) => {
+                    tracing::warn!("Could not construct a glob for {}: {}, please make sure your path contains no *", framework_path, err);
+                    tracing::warn!("This path will not be scanned for dylibs!");
+                    continue;
+                }
+            };
+
+            let files = files.filter_map(|p| match p {
+                Ok(v) => Some(v),
+                Err(err) => {
+                    tracing::warn!(
+                        "Error while globbing for Mach-O files in {}: {}",
+                        framework_path,
+                        err
+                    );
+                    None
+                }
+            });
```

Comment:
We can just use flatten, don't think we need log these errors, we can just skip them
```suggestion
            let files = files.flatten();
```